creation/interfaces-and-networks.md

```diff
@@ -692,3 +687,209 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
 > into VMI guest. Because of that, when configuring SR-IOV operator
 > policies, make sure you define a pool of VF resources that uses
 > `driver: vfio`.
+
+Once the operator is deployed, an [SriovNetworkNodePolicy
+](https://github.com/openshift/sriov-network-operator#sriovnetworknodeconfigpolicy)
+must be provisioned, in which the list of SR-IOV devices to expose (with
+respective configurations) is defined.
+
+Please refer to the following `SriovNetworkNodePolicy` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 1500
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov_net
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriov_net`.
+
+It will also render an associated `SriovNetworkNodeState`, which looks like:
+
+```
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+  metadata:
+    name: sriov-worker
+    namespace: sriov-network-operator
+    ownerReferences:
+    - apiVersion: sriovnetwork.openshift.io/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SriovNetworkNodePolicy
+      name: default
+  spec:
+    interfaces:
+    - mtu: 1500
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      vfGroups:
+      - deviceType: vfio-pci
+        resourceName: sriov_net
+        vfRange: 0-7
+  status:
+    interfaces:
+    - Vfs:
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+      - deviceID: "1016"
```

Comment:
You can just omit the rest of the list with three dots to save space in the user guide.

Reply:
Done.

Reply:
Done.